hedera-mirror-rest/controllers/contractController.js

```diff
@@ -1342,8 +1394,9 @@ if (utils.isTestEnv()) {
       extractSqlFromContractFilters,
       extractTimestampConditionsFromContractFilters,
       formatContractRow,
-      getContractByIdOrAddressQuery,
+      getContractByIdOrAddressQuery: getContractByIdOrAddressContractEntityQuery,
```

Comment:
```suggestion
     getContractByIdOrAddressContractEntityQuery,
```

and change references in other modules